components/FoundItem.js

```diff
@@ -0,0 +1,31 @@
+import React from "react";
+import Image from "next/image";
+import styled from "styled-components";
+
+export default function FoundItem({ name, description, img, id }) {
+  return (
+    <div>
+      <li>
+        This item is: {name}. Its description is: {description}. Its ID is: {id}
+        . It looks like:
+        <ImageContainer>
+          <Image
+            objectFit="cover"
+            src={img}
+            width={200}
+            heigth={200}
+            layout="fill"
+            alt="lost items"
```

Comment:
I would suggest the alt text is "found items" because it's the found items component?